tests/unit/todo-footer/TodoFooter.spec.js

```diff
@@ -0,0 +1,13 @@
+import { shallowMount } from '@vue/test-utils'
+import TodoFooter from '@/components/todo-footer/TodoFooter.vue'
+
+describe('TodoFooter component tests', () => {
+  const wrapper = shallowMount(TodoFooter, {
+    mocks: {
+      $t: msg => msg
+    }
```

Comment:
It is not necessary to mock the translation function as we are not using internationalization